vendor/modules.txt

```diff
@@ -189,7 +189,11 @@ github.com/inconshreveable/mousetrap
 github.com/json-iterator/go
 # github.com/konsorten/go-windows-terminal-sequences v1.0.3
 github.com/konsorten/go-windows-terminal-sequences
+<<<<<<< HEAD
```

Comment:
Rebasing on `main` should clear these up

Reply:
Huh turns out it didn't. If you delete just the conflict lines, and then run `go mod vendor`, it should fix all of this

Reply:
I dont see this anymore on my branch?

Reply:
Seems gone now? /shrug